pitch.md

```diff
@@ -1,60 +1,224 @@
+
+I will admit, a language that you type out is not the optimal way to compose
+the cues; a GUI paired with the DSL, would bring this project to a level who's
+convenience and ease of use would make it practical solution for stage managers
+and designers.
 
 ## Why you?
 _What excites you about this idea? How did you come up with it?_
 
+As a stage manager who has worked on several large productions involving hundreds
+of cues, I've always dreaded transferring designer cue sheets' cues into the
+prompt book. I've also longed for a way to be able to search and jump to cues
+without spending time searching through the script.
+
+For the past several shows I've worked on, I've debated making a digital prompt
+book, but was dissapointed with the current solutions&mdash;annotating a PDF.
+With this frustration, I've debated coming up with my own solution and this
+project has presented a good opportunity to explore it further.
 
 ## Interface (syntax)
 _What kinds of things can the user say in your language. How, generally, do they
 say it? **Be as specific as possible.** You don't need to design syntax; but you
 should be more specific than "The user describes properties of unicorns.", e.g.,
-by describing which unicorn properties (horn color, tattoo, etc.) are most 
-important to users and what kinds of values those properties might have._ 
-
+by describing which unicorn properties (horn color, tattoo, etc.) are most
+important to users and what kinds of values those properties might have._
+
+The user should be able to **describe a variety of cue types** (e.g. lighting,
+sound, projection, as well as custom cue types) and **specify** with varying degrees
+of exactness **where the cue lives** in the script (e.g. top of a page, at a specific
+line or word, etc.). The user should also be able to **assign each cue a number
+and state**: many times stage managers will right in a *standby* for cue X as well
+as a *GO* call for cue X. There should also be the ability to **group cues to go
+at once**; for example, if one wanted to cue lights and sound at the same time.
+The cue notation should allow for comments/notes; however, it should always be
+preferred to express patterns within the languge itself. In other words, since
+calling a cue with a visual prompt is a common pattern, it would be best to express
+this with a language keyword as opposed to simply leaving this in a comment.
 
 ## Operation (semantics)
 _What might happen when a program runs? Is there a computational model that
 corresponds with your domain? How does a program interact with the
-user? What kinds of semantic (i.e., non-syntax) errors might occur, and how 
+user? What kinds of semantic (i.e., non-syntax) errors might occur, and how
 might they be communicated to the user?_
 
+This DSL fits under the computational model of a **data interpretation** since the
+language simply describes the positions and relationship of cues to one another.
```

Comment:
Simultaneity is the main link and sometimes a fixed time. In an alternate world, I'd imagine being able to describe dependencies between cues so that if one if missed, there would be a programmatic way of determining what to do next; however, I think this would be inappropriate for the end-users and generally more complicated than people desire. Often times, live decisions need to made.